src/kit.ts

```diff
@@ -798,17 +800,31 @@ async function scanDirForClangForMSVCKits(dir: string, vsInstalls: VSInstallatio
             const vs_arch = (version.target && version.target.triple.includes('i686-pc')) ? 'x86' : 'amd64';
 
             const clangArch = (vs_arch === "amd64") ? "x64\\" : "";
-            const clangKitName = `Clang ${version.version} ${clang_cli} for MSVC ${vs.installationVersion} (${install_name} - ${vs_arch})`;
-            if (binPath.startsWith(`${vs.installationPath}\\VC\\Tools\\Llvm\\${clangArch}bin`) && await util.checkFileExists(util.lightNormalizePath(binPath))) {
-                clangKits.push({
-                    name: clangKitName,
-                    visualStudio: kitVSName(vs),
-                    visualStudioArchitecture: vs_arch,
-                    compilers: {
-                        C: binPath,
-                        CXX: binPath
-                    }
-                });
+            const useClangVersion: boolean = versionLess(version.version, '14.0.0');
+            const clangKitName: string = (isClangCL ? `Clang-cl` : `Clang`) + (useClangVersion ? ` ${version.version}` : "") + ` ${clang_cli} for ${install_name} - ${vs_arch} (MSVC ${vs.installationVersion})`;
```

Comment:
I think we should keep the format of the kit name the same for now so that duplicate kits don't get generated.